src/amo/components/SearchSort/SearchSort.scss

```diff
@@ -65,6 +73,10 @@
     background: transparentize($link-color, 0.8);
     box-shadow: none;
   }
+
+  @include respond-to(large) {
```

Comment:
The issue with that is especially as we're using viewports that aren't tied to device sizes but just sane breakpoints... you'd end up without content when you turned your phone around.

In general I'd say "we should only load important content and if we're hiding a LOT of stuff entirely except desktop... yes maybe it isn't needed". But unless we can say a specific component should only load with a certain UA... I kinda dislike loading different sets of content. Take, for example, the search page on a Nexus 6 in portrait versus landscape:

### Portrait
![screen shot 2017-06-21 at 14 12 21](https://user-images.githubusercontent.com/90871/27385835-cd1d95e6-568b-11e7-95ab-dc1b3422416f.png)

### Landscape
![screen shot 2017-06-21 at 14 12 24](https://user-images.githubusercontent.com/90871/27385842-d22110ae-568b-11e7-9ed4-86baea6b38e6.png)

### Landscape (w/o content loaded)
![screen shot 2017-06-21 at 14 13 06](https://user-images.githubusercontent.com/90871/27385854-dc63a536-568b-11e7-8d2c-416713cefd14.png)

----

I think there's enough room to show the content and furthermore we should load it to prevent weirdness with different UAs getting/not getting content. I think that will make things harder to QA/test and debug, in addition to being a worse UX.